src/main/java/com/yahoo/bullet/rest/service/PreprocessingService.java

```diff
@@ -21,6 +24,17 @@
     private static final String WINDOW_KEY_STRING = "window";
     private static final BulletQueryBuilder QUERY_BUILDER = new BulletQueryBuilder(new BulletConfig());
     private static final Gson GSON = new GsonBuilder().create();
+    private int maxConcurrentQueries;
```

Comment:
Write as:
@Value("${bullet.max.concurrent.queries}")
private int maxConcurrentQueries;

Then you don't need the constructor below.